### ISSUE TITLE
[BUG] Header of Action Tab Misaligned on Tidy5eSheet

### ISSUE THREAD

**Issue by Red5MT** [eastcw/foundryvtt-dnd5eCharacterActions#8]
When using Tidy5eSheet, the header of th action tab is misaligned. "Range", "Hit/DC", and "Damage" are shifted to the right.

While Tidy5eSheet is not specifically called out in the description of this module, it does seam to be a widely used character sheet. The Action Tab module works fine with the defualt 5e character sheet as well as Compact D&D Beyond sheet. It would be nice if it were to work cleanly with Tidy5eSheet as well. Hopefully it would not be a difficult adjustment.

Thank you for making the tab for character sheets in the first place. It is surprising that it wasn't part of the default sheet in the first place.
![Header](https://github.com/eastcw/foundryvtt-dnd5eCharacterActions/assets/83143296/cd4e224b-71d6-4ba0-9390-4e610b8e75b3)

**Comment by AJCraddock** [comment]
I think I figured out the issue. Instead of setting the last column to display none they should be set to min/max-width. I am going to try and put out a PR.

original
```css
.tidy5e-sheet .character-actions-dnd5e .items-header .item-controls {
  display: none;
}
```
my attempt at a solution: 
```css
/* Had to use !important to get past a weird rule done by I am guessing tidy5e sheet*/

.character-actions-dnd5e.items-list.inventory-list .items-header .item-controls, 
.tidy5e-sheet .character-actions-dnd5e .item-controls.context-menu {
    min-width: 30px !important;
    max-width: 30px !important;
    justify-content: center !important;
}
```

![image](https://github.com/eastcw/foundryvtt-dnd5eCharacterActions/assets/5339435/3a1b5ad3-398b-4cff-a152-476ac5050688)

**Comment by eastcw** [comment]
Sorry I've been pretty slow on this and doing other stuff with my life. Is this still an issue? I can't reproduce it in the latest version of tidy5esheet, since the new version implements its own version of the actions tab.

**Comment by Red5MT** [comment]
I can confirm that it is an issue in D&D5e 2.4.1. I have not moved on to 3.0 due to some othe rmodules.

It is certainly understandable if you decide to not update this verision of Header Action Tab for this particular issue. It was not originally supported in any case.

**Comment by eastcw** [comment]
I've published v6.1.2 that should fix this issue, and will hopefully be the last release supporting 5e 2.4.x before updating the module for 5e 3.0.x going forward.

**Comment by Red5MT** [comment]
Works great! We 2.4.1 hang-er-ons appreciate the effort.
